dags/inference/jetstream_pytorch_inference.py

```diff
@@ -65,7 +65,7 @@
           "checkpoint_quantized": "gs://inference-benchmarks/models/llama2-13b-chat/pytorch/llama-2-13b-chat-merged-int8-per-channel",
           "dataset": "openorca",
           "tokenizer": "tokenizer.llama2",
-          "batch_sizes": [96],
+          "batch_sizes": [8, 32, 96],
```

Comment:
Add  batch_size `64` please. This will provide intermittent data points for plotting latency/throughput curve

Reply:
added